thrones/config/routes.rb

```diff
@@ -0,0 +1,6 @@
+Rails.application.routes.draw do
+    root 'houses#index'
+    resources :characters
+    resources :houses
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```

Comment:
Nice work using resources! Think you're there now, but think the user flow would work much better here if you nest your characters inside the houses.